src/algorithms/reco/JetReconstruction.cc

```diff
@@ -1,13 +1,14 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
-// Copyright (C) 2023 Derek Anderson, Zhongling Ji
+// Copyright (C) 2024 Derek Anderson, Zhongling Ji, Dmitry Kalinkin
 
 // class definition
 #include "JetReconstruction.h"
 
 // for error handling
 #include <JANA/JException.h>
-#include <Math/GenVector/LorentzVector.h>
 #include <edm4hep/Vector3f.h>
+#include <edm4hep/MCParticleCollection.h>
```

Comment:
```suggestion
#include <edm4hep/MCParticleCollection.h>// IWYU pragma: keep
```
Ref: https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYUIsDifficult.md#handling-template-arguments